gremlin-dotnet/src/Gremlin.Net/Driver/ResultSet.cs

```diff
@@ -0,0 +1,85 @@
+#region License
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#endregion
+using Gremlin.Net.Driver.Exceptions;
+using System.Collections;
+using System.Collections.Generic;
+namespace Gremlin.Net.Driver
+{
+    /// <summary>
+    /// A ResultSet is returned from the submission of a Gremlin script to the server and represents the results provided by the server
```

Comment:
Same here as with `IResultSet`. You could also use `<inheritdoc />` to avoid the duplication.

Reply:
Fixed